.github/workflows/ci.yml

```diff
@@ -401,7 +395,7 @@ jobs:
   build_test_runtime:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_test_runtime')
-    runs-on: ubuntu-20.04-64core
+    runs-on: ubuntu-20.04
```

Comment:
2 minutes -> 7 minutes for this job. 

* before: https://github.com/iree-org/iree/actions/runs/8806894791/job/24172720239
* after: https://github.com/iree-org/iree/actions/runs/8806704450/job/24172130260?pr=17145
  * Around 3 minutes linking - faster than Windows but still slow